Content.Shared/Anomaly/SharedAnomalySystem.cs

```diff
@@ -117,7 +117,8 @@ public void RefreshPulseTimer(EntityUid uid, AnomalyComponent? component = null)
     /// Begins the animation for going supercritical
     /// </summary>
     /// <param name="uid"></param>
-    public void StartSupercriticalEvent(EntityUid uid)
+    /// <param name="component"></param>
+    public void StartSupercriticalEvent(EntityUid uid, AnomalyComponent? component = null)
```

Comment:
Do this the modern, `Entity<T>` way:
```suggestion
    public void StartSupercriticalEvent(Entity<AnomalyComponent?> ent)
```
You'll need to replace `uid` with `ent` or `ent.Owner`, and `component` with `ent.Comp` in the rest of this method.